gem_rspec/install_generator_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+require 'generator_spec'
+require 'generators/smashing_documentation/install_generator'
+RSpec.describe SmashingDocumentation::Generators::InstallGenerator, type: :generator do
+  let(:rails_helper) { "spec/rails_helper.rb" }
+  let(:spec_helper) { "spec/spec_helper.rb" }
+  let(:docs_template) { "smashing_docs/template.md" }
+  context "when rspec is installed" do
+    describe "#update_spec_helper" do
+      it "appends the SmashingDocs config to rails_helper.rb" do
+        run_generator
+        expect(File).to exist(rails_helper)
+        expect(File.read(rails_helper)).to include("SmashingDocs.config do")
+      end
+    end
+
+    describe "#update_spec_helper" do
```

Comment:
This describe has the same name as the one above it. You can probably combine them